docs/cpp/config-clang-mac.md

```diff
@@ -3,124 +3,143 @@ Order: 4
 
-## Add VS Code to your PATH
+    ![C/C++ extension](images/cpp/cpp-extension.png)
 
-In order to start VS Code from the command line, we'll need to add it to the PATH environment variable. This only needs to be done once, on first use.
+### Ensure Clang is installed
 
-1. Open VS Code
-1. Press `kb(workbench.action.showCommands)` to open the Command Palette.
-1. Start typing "Shell" and from the list of suggestions choose **Shell Command: Install 'code' command in PATH**.
+Clang may already be installed on your Mac. To verify that it is, open a macOS Terminal window and enter the following command:
 
-   ![Shell command in Command Palette](images/clang-mac/mac-command-palette-shell-command.png)
+```bash
+clang --version
+```
 
-1. You should see a notification in the lower right of the VS Code window that tells you that VS Code was successfully added to the PATH.
-1. Close VS Code.
+1. If Clang isn't installed, enter the following command to install the command line developer tools:
 
-## Start VS Code in a folder
+```bash
+xcode-select --install
+```
 
-In the macOS Terminal, create an empty folder called "projects" and then a subfolder called "helloworld". Navigate into it, and open VS Code (`code`) in that folder (`.`) by entering the following commands:
+## Create Hello World
 
-```bash
+From the MacOS Terminal, create an empty folder called `projects` where you can store all your VS Code projects, then create a subfolder called `helloworld`, navigate into it, and open VS Code in that folder by entering the following commands:
+
+```cmd
 mkdir projects
 cd projects
 mkdir helloworld
 cd helloworld
 code .
 ```
 
-The **code .** command opens VS Code in the current working folder, which becomes your *workspace*. Before we can get IntelliSense support, or compile and debug our code, we have to configure VS Code to use Clang/LLDB. After completing the configuration, we will have three files in a `.vscode` subfolder:
+The `code .` command opens VS Code in the current working folder, which becomes your "workspace". As you go through the tutorial, you will create three files in a `.vscode` folder in the workspace:
 
-- `c_cpp_properties.json` (compiler path and IntelliSense settings)
-- `tasks.json` (build instructions)
+- `tasks.json` (compiler build settings)
 - `launch.json` (debugger settings)
+- `c_cpp_properties.json` (compiler path and IntelliSense settings)
 
-To reuse the configuration, you can copy these files to a new workspace and adjust the program name and other settings as needed.
+### Add hello world source code file
 
-## Configure the compiler path
+In the File Explorer title bar, select **New File** and name the file `helloworld.cpp`.
 
-1. Press `kb(workbench.action.showCommands)` to open the Command Palette. It looks like this:
+![New File title bar button](images/msvc/new-file-button.png)
```

Comment:
You may want take new screenshots since the Debug/Run icon has changed and the top menu now show "Run" instead of "Debug". You could also update the other tutorials for this change. It can also happen on a later PR.

Reply:
fixed - trimmed the debug/run button out since it isn't germane to the screenshot.